src/activities/components/ActivityList.vue

```diff
@@ -426,6 +430,40 @@ export default {
       this.slots = 'all'
       this.type = 'all'
     },
+    checkArchivedTypes () {
+      const archivedTypes = {}
+
+      for (const activityType in this.activityListTypeFilter) {
+        // count the number of archived types and store in a hashmap, defaulting to zero
+        if (this.activityListTypeFilter[activityType].status === 'archived') {
+          archivedTypes[this.activityListTypeFilter[activityType].id] = 0
+        }
+      }
+
+      console.log(this.activities)
+
+      for (const num in this.activities) {
```

Comment:
You can use a for/of loop here, e.g.:

```js
for (const activity of this.activities) {
```

and saves the line `const activity = this.activities[num]` below.